bin/node/src/commands/store.rs

```diff
@@ -147,15 +147,15 @@ impl StoreCommand {
         Store::bootstrap(genesis_state, data_directory)
     }
 
-    fn generate_genesis_account() -> anyhow::Result<AccountFile> {
+    pub fn generate_genesis_account() -> anyhow::Result<AccountFile> {
```

Comment:
Probably doesn't need to be `pub`?

```suggestion
    fn generate_genesis_account() -> anyhow::Result<AccountFile> {
```

Reply:
Yea forgot I left that there